Add content to Producet_Management.md

zhangchengyue opened the pull request:
In the Producet_Management.md file, I added some key points on product management, then talks about how CSC301 students can take action on them, and also talks about the difference between managing platforms and products. A link to a book (free copy) is also included as extra resource for those who are interested.

zhangchengyue committed:
producet management md

SanjanaGirish commented on Topics/Product_Management.md:
```diff
@@ -1 +1,39 @@
+* __**Coordinating Development:**__ Assuming they have received a green light to move forward with their product’s strategic plan, coordinate with the relevant teams—product marketing, development, etc.—to begin executing the plan.
+* __**Acting on Feedback and Data Analysis:**__ Finally, after building, testing, and introducing the product to the marketplace, learning via data analysis and soliciting direct feedback from users, what works, what doesn’t, and what to add. Working with the relevant teams to incorporate this feedback into future product iterations.
+
+### How should CSC301 students perform product management?
```
Very useful insight for CSC301 students, great job! The instructions for utilizing product management strategies are neatly tied with the above info on what product managers do. It would be nice to also point to David's lecture from Week 2 on product management.

SanjanaGirish reviewed (changes requested):
Overall great job! The topic has been explained in-depth and I really enjoyed the embedded links to LinkedIn articles and websites for further reading. Please make sure you update the Readme to incorporate your subtopic.

Another suggestion would be to keep the overall theme/topic in mind while explaining. Defining related terms can take the focus away from the main message on 'Resources for Product Management'.

zhangchengyue committed:
Modified based on TA's comment

zhangchengyue commented:
I have modified the page based on TA's comment. I added more resources mentioned in Week 2 lecture, and deleted some of the concepts defined that may distract the focus of the resources.

SanjanaGirish reviewed (changes requested):
Great incorporation of feedback. Please resolve merge conflicts with the main branch and update Readme. It would also be easier to resolve conflicts if you make your contributions in a separate file and link the file in Product_Management.md

zhangchengyue committed:
updated readme

zhangchengyue committed:
Merge branch 'main' into A3_branch
